### ISSUE TITLE
<Form Wizard Redesign for Ch 31 and Ch 36> from <Benefits Non-Disability Experience>

### ISSUE THREAD

**Issue by steele-lm** [department-of-veterans-affairs/va.gov-team#73341]
# Content, accessibility, information architecture (CAIA) new initiative collaboration request

Use this ticket to start collaboration on a new initiative with the sitewide content, accessibility, and information architecture (CAIA) team.

**Note:** If you’re already partnering with us on an initiative, you don't need to fill out this request form. If you don’t have access to your initiative’s CAIA epic, post a message in the [#sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-acessibility-ia) slack channel and tag `@Terry Nichols` and `@Lily Strelich`.

## About your team

- Team name: Benefits and Claims | Lifestage Benefits Crew | Non-Disability Experience
- OCTO-DE product owner: Emily Theis
- Product manager: Laura Steele
- Designer: Fiorella Glavez
- Accessibility specialist: none
- Content writer: none
- Slack channel: benefits-non-disability
 
### Which of these descriptions best fits the work we’ll partner on?
Select all that apply.
- [ ] Digitizing a new form
- [ ] Creating a new digital tool
- [x] Updating an existing form or tool
- [ ] Translating a form or tool into Spanish
- [ ] Adding new unauthenticated content to VA.gov
- [ ] Updating existing unauthenticated content on VA.gov
- [ ] Something else:  `insert description`

### What's the nature of your initiative and desired outcomes?

An accessibility audit flagged some issues with the start wizard that is used by Chapter 36 (25-8832) and Chapter 31 (28-1900) forms, so we've redesigned the wizard experience. A11y tickets:
#64692
#64688
#64218 
#63702 

## Collaboration timeframe

**Note:** We work on nearly every OCTO product and manage all unauthenticated content on VA.gov, so we will need to prioritize intake requests based on overall workload and VA and OCTO priorities.

### Is this work tied to a Congressional mandate, change in law or policy, or upcoming event with a specific deadline?
- [ ] Yes
- [x] No
- [ ] I’m not sure

### Where are you at in your timeline?
Tell us briefly about what you're working on now (such as initial discovery, wireframing, or usability research planning) and add any known dates for upcoming milestones or deadlines.

- We have created new designs for the wizard experience, and we need to determine what level of CAIA review is needed and if these changes require participation in the collab cycle.

### Which phases of the collaboration cycle have you completed?
Select all that apply.
- [ ] Design intent 
- [ ] Midpoint review 
- [ ] Staging 
- [x] None. Collab cycle participation TBD.

### Collaboration cycle ticket 

If you’re going through the collaboration cycle, provide your ticket number and link:
- **Ticket Number:** `xxxxxx`

## Supporting artifacts
Provide links to any supporting artifacts that can help us better understand your initiative and begin collaboration. Include artifacts like your product outline, user flows, mockups and prototypes, or any draft content.
- [Existing flow of the start wizard for Ch 31](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1684161597392/5c1f8869cf5d03b792c8b82cf19b39dc26c486c2?wid=0-1703022981131)
- [New design for the start page of Ch 31](https://www.figma.com/file/gqBtkNOCslhEwes7TsBaQy/Ch31-VR%26E--Redesign?type=design&node-id=2%3A22844&mode=design&t=kwTwaVKFR8BCGMeA-1)
- [Existing flow of Ch 31](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1691434304975/3f0c405c820c184d21393b3bbd76324a5ee7c677?sender=u0e8ac1d6d7681bb7e1b80558)
- [New design for start page of Ch 36](https://www.figma.com/file/DEnVQXRrNvwJ3UkKkYYJ1Q/Ch-36-Redesign?type=design&node-id=1%3A4&mode=design&t=niClq2OBhoN1NUXl-1)
- Designs for skipping the wizard for in-progress Ch 31 forms:
   - [How to apply](https://www.figma.com/file/gqBtkNOCslhEwes7TsBaQy/Ch31-VR%26E--Redesign?type=design&node-id=128%3A8585&mode=design&t=p6wBtQbVHexIKh2h-1)
   - [Eligibility](https://www.figma.com/file/gqBtkNOCslhEwes7TsBaQy/Ch31-VR%26E--Redesign?type=design&node-id=128%3A9534&mode=design&t=p6wBtQbVHexIKh2h-1)
   - [Reemployment](https://www.figma.com/file/gqBtkNOCslhEwes7TsBaQy/Ch31-VR%26E--Redesign?type=design&node-id=128%3A9797&mode=design&t=p6wBtQbVHexIKh2h-1)
   - [Rapid access to employment](https://www.figma.com/file/gqBtkNOCslhEwes7TsBaQy/Ch31-VR%26E--Redesign?type=design&node-id=128%3A10066&mode=design&t=p6wBtQbVHexIKh2h-1)
   - [Self-employment](https://www.figma.com/file/gqBtkNOCslhEwes7TsBaQy/Ch31-VR%26E--Redesign?type=design&node-id=128%3A10334&mode=design&t=p6wBtQbVHexIKh2h-1)
   - [Long-term services](https://www.figma.com/file/gqBtkNOCslhEwes7TsBaQy/Ch31-VR%26E--Redesign?type=design&node-id=128%3A10602&mode=design&t=p6wBtQbVHexIKh2h-1)
   - [Independent living](https://www.figma.com/file/gqBtkNOCslhEwes7TsBaQy/Ch31-VR%26E--Redesign?type=design&node-id=128%3A10870&mode=design&t=p6wBtQbVHexIKh2h-1)

## Next steps
- [ ] Post a link to this issue in the [#sitewide-content-sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-accessibility-ia) slack channel and tag `@Terry Nichols`. We’ll reply within 24 hours to acknowledge receipt. We'll then review the artifacts in more detail and work with you to determine timing for collaboration. Depending on the work, we may schedule a kickoff meeting to better understand how we'll partner together.
- [ ] Review the remaining next steps in the **VFS team tasks** section here.

## Tasks

<br>

```[tasklist]
### VFS team tasks
- [ ] **If you also need engineering support from the public websites team**, fill out their intake request form as well.<br> [Open a public websites intake request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E)
- [ ] **If you're requesting a page/URL redirect, a URL change, or a vanity URL**, submit an additional request for this work. <br> [Open a redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=jennymayoco%2C+jilladams%2C+RLHecht&labels=sitewide+CAIA%2C+Sitewide+IA%2C+Public+Websites%2C+VA.gov+frontend%2C+Redirect+request&projects=&template=redirect-request.md&title=Redirect+Request)
- [ ] **If you're creating an experimental design**, also contact the design system team. <br> [Read more about experimental designs](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) <br> [Suggest an addition or update to the design system team](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)
- [ ] **If you're requesting a11y support for usability research**, submit a research support request next. <br> [Open an a11y research support request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name)
```

```[tasklist]
### CAIA operations
- [ ] Initial Review - @coforma-terry and @strelichl 
```

```[tasklist]
### CAIA a11y tasks
```

```[tasklist]
### CAIA content tasks
```

```[tasklist]
### CAIA IA tasks
```

**Comment by coforma-terry** [comment]
@sara-amanda  - I have added the linked tickets as tasks under the a11y section for this 👍

**Comment by steele-lm** [comment]
@jstrothman

**Comment by steele-lm** [comment]
@coforma-terry I think this ticket can be closed bc we all agreed that the wizards will be removed once we start updating the forms.

**Comment by aprocik1** [comment]
I'm tagging in @strelichl, since Terry is no longer working on CAIA.
